includes/pages/user_shifts.php

```diff
@@ -43,18 +43,21 @@ function update_ShiftsFilter_timerange(ShiftsFilter $shiftsFilter, $days) {
   $start_time = $shiftsFilter->getStartTime();
   if ($start_time == null) {
     $start_time = time();
+    if($start_time < 1495238400) {
+      $start_time = 1495238400; // 20.05.2017 00:00
+    }
   }
   
   $end_time = $shiftsFilter->getEndTime();
   if ($end_time == null) {
-    $end_time = $start_time + 24 * 60 * 60;
+    $end_time = $start_time + 12 * 24 * 60 * 60;
```

Comment:
No, this is the default end time for the shifts view. By default all shifts between 2017-05-20 and 2017-05-30 are shown now.